packages/compose/src/hooks/use-keyboard-shortcut/index.js

```diff
@@ -37,14 +37,18 @@ function isAppleOS( _window = window ) {
 	);
 }
 
+/* eslint-disable jsdoc/valid-types */
 /**
  * Attach a keyboard shortcut handler.
  *
- * @param {string[]|string}         shortcuts  Keyboard Shortcuts.
- * @param {Function}                callback   Shortcut callback.
- * @param {WPKeyboardShortcutConfig} options    Shortcut options.
+ * @see https://craig.is/killing/mice#api.bind for information about the `callback` parameter.
+ *
+ * @param {string[]|string} shortcuts Keyboard Shortcuts.
+ * @param {import('mousetrap').MousetrapInstance['bind']} callback Shortcut callback.
```

Comment:
Following [this comment](https://github.com/WordPress/gutenberg/pull/32168/commits/44f9a5012f9f6ce3044b17a9db6d2a191df9957a#r638647921), shouldn't this be `(e: ExtendedKeyboardEvent, combo: string) => any` too?

Reply:
Nevermind, somehow GitHub didn't load the most recent version of the PR 🤷